src/shogun/kernel/MaternKernel.cpp

```diff
@@ -0,0 +1,198 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Gil Hoben
+ */
+
+#include <shogun/distance/EuclideanDistance.h>
+#include <shogun/kernel/MaternKernel.h>
+
+using namespace shogun;
+
+MaternKernel::MaternKernel() : ShiftInvariantKernel()
+{
+	auto distance = std::make_shared<EuclideanDistance>();
+	distance->set_disable_sqrt(false);
```

Comment:
@karlnapf I turned this off, but I guess it could be true for the sake of efficiency?

Reply:
what is that for again?

Reply:
It’s to avoid computing the square root of the Euclidean distance